src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -172,21 +253,10 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
             builder.field("default_operator", operator.name().toLowerCase(Locale.ROOT));
         }
 
-        if (lowercaseExpandedTerms != null) {
-            builder.field("lowercase_expanded_terms", lowercaseExpandedTerms);
-        }
-
-        if (lenient != null) {
-            builder.field("lenient", lenient);
-        }
-
-        if (analyzeWildcard != null) {
-            builder.field("analyze_wildcard", analyzeWildcard);
-        }
-
-        if (locale != null) {
-            builder.field("locale", locale.toString());
-        }
+        builder.field("lowercase_expanded_terms", settings.lowercaseExpandedTerms());
```

Comment:
we still have to be careful with users potentially setting stuff to null explicitly I guess, hence leave the null checks or reject them at set time or perform some kind of validation.